src/components/UserCard.tsx

```diff
@@ -14,6 +14,12 @@ import { UserCardType } from "../types/UserCard";
 import UserModal from "./UserModal";
 import Avatars from "../definitions/Avatars";
 
+const commitmentLevelColors: Record<string, string> = {
+  Low: "red.400",
+  Medium: "yellow.400",
+  High: "green.400",
+};
+
```

Comment:
Isn't of defining this in two places, let's move this to [this](https://github.com/HackGT/match/blob/ca9f5edc6622de1e78d3b69db97646ce0d989e41/src/definitions/CommitmentLevels.tsx) file and just import it in both files